package.json

```diff
@@ -53,7 +53,9 @@
     "winston-elasticsearch": "^0.5.0",
     "xml2js": "^0.4.16",
     "youtube-api": "^2.0.2",
-    "youtube-dl": "^1.11.1"
+    "youtube-dl": "^1.11.1",
+    "lodash": "^4.17.4",
```

Comment:
Is requiring lodash for such a niche use case necessary? Most of lodash's tools are just shorthands for basic JavaScript logic, and that should preferably be used.

Reply:
Not sure what the project's philosophy is, but basic javascript is kind of unreliable in the longterm, this simplifies code and if used consistently makes it less brittle. Definitely can be removed though.

Reply:
The usage of lodash especially in these cases is almost useless since it's just such basic JS. For instance, your usage of `_.head(res.body)` is just going to be `res.body[0]` if I'm not mistaken. As for `_.startCase(from)`, I don't know how necessary it is either as it's just letter changing.